seqr/views/utils/variant_utils.py

```diff
@@ -91,6 +92,8 @@ def _saved_variant_genes_transcripts(variants):
             for gene_id, transcripts in var.get('transcripts', {}).items():
                 gene_ids.add(gene_id)
                 transcript_ids.update([t['transcriptId'] for t in transcripts if t.get('transcriptId')])
+            for family_guid in var['familyGuids']:
+                family_genes[family_guid].update(var.get('transcripts', {}).keys())
```

Comment:
Do we need to include the families who have no transcripts?

Reply:
yes overall, as this is used for the the list of families for filtering which projects to return etc. However, I can add an extra filter before we get the family list for rna data, as theres no reason to include rna data for families with no genes

Reply:
added this below